core/templates/dev/head/player/PlayerServices.js

```diff
@@ -302,6 +307,43 @@ oppia.factory('oppiaPlayerService', [
     }
   };
 
+  var _loadInitialInformationCardData = function(successCallback) {
+
+      var infoCardDataUrl = '/createhandler/data/' +  _explorationId;
+      var infoCardSnapshotsUrl = '/createhandler/snapshots/' +  _explorationId;
+      var infocardStatisticsUrl = '/createhandler/statistics/' +  _explorationId + '/all';
+
+      // This is needed to get exploration objective/goal,contributors. 
+      // and exploration tags 
+      $http.get(infoCardDataUrl).success(function(data) {
+        _explorationObjective = data.objective;
+        _explorationTags = data.tags;
+        _explorationContributors = data.rights.owner_names.concat(data.rights.editor_names);
```

Comment:
Add TODO:

  // TODO(sll): compute the actual contributors based on the list of commits in the backend